### PR TITLE
Repo setup

### PR DESCRIPTION
**Pull request by jbaxleyiii**
Initial basic setup of the repo with testing and release of artifiacts

### CODE REVIEW

File: graphql/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[package]
```
Comment (queerviolet) [inline graphql/Cargo.toml:1]: Maybe let's leave it out till we figure out how we want to pull in the existing graphql-parser? From my limited experimenting with it, it was easiest to treat it as an external crate even if we fork it into this repo

**Comment by queerviolet** [comment]
Ah, one other comment: could we include getting started instructions in `README.md`? Maybe it's just bc I'm new to Rust and could use the help, but I find it odd that every js project includes the same `npm install && npm start` instructions, but in Rustland, where the build process is not necessarily as obvious (to me?), we don't seem to do that?

File: graphql/Cargo.toml (hunk 1 above)
Comment (jbaxleyiii) [reply graphql/Cargo.toml:1]: @queerviolet yep that sounds good to me. I mainly wanted to show that workspaces / multi crates was easy to setup. Happy to remove!

Comment (queerviolet) [reply graphql/Cargo.toml]: It is good to see, I hadn't known about this feature, actually!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,51 @@
...
+
+```sh
+cargo 1.42.0 (86334295e 2020-01-31)
+```
+
+### Running the project
+To run the project locally, you can use `cargo` to build and exexecute the bin of the `apollo` crate.
+
+```
+cargo run --bin apollo
```
Comment (queerviolet) [inline README.md:26]: Let's put `cargo run` right up top, akin to the `npm install && npm run` part of so many JS readmes.

Also, if we get rid of the `graphql` crate—or make it a library crate, I think?—we won't have to specify the `--bin apollo`, we can just do `cargo run`

Comment (jbaxleyiii) [reply README.md:26]: @queerviolet we can get rid of the `--bin apollo` but I suspect we will want another crate pretty quickly into this build out. I'll add a section about running the CLI as well. I suspect there is a way to make only one of the packages a "bin" command to ease this as well

Comment (jbaxleyiii) [reply README.md]: @queerviolet yep! That worked great :)

**Comment by jbaxleyiii** [comment]
@queerviolet @JakeDawkins are you okay with us merging this?

**Comment by JakeDawkins** [comment]
@jbaxleyiii Yep! This looks like a fantastic start :)

**Pull request merged**